### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by Bumblebee001** [GrandOrgue/grandorgue#1871] [General]
Voicing:

Is it possible to by-pass the need to "Apply" changes whilst voicing? Proper voicing can only be achieved best by adjusting and listening to the changes as they are being made - on-the-fly.

As I had occasion to request in the past, sliders would be more practical and optically meaningful than scrolling numbers up and down clicking on tiny arrows.

I am still optimistic about having a "Brightness" feature that can be changed in addition to Amplitude, Gain and Tuning.

Flexible Wind:

I have mentioned this desirable feature but I recall no response to it. This feature causes subtle variations in the sound to emulate real-life variations in wind flow through pipes adding not just realism but may actually put less stress on our hearing. Pure sounds tire the ears subconsciously and will force the player to stop playing after a brief period. 

Can these be given their due importance. I believe that these features would be in themselves game changers and their value should not to be underestimated.

**Comment by Psi58** [comment]
Replying on the basis that any feedback is better than none (even if from a non-expert beginner...)

1) Real-time voicing adjustment using sliders : agreed, this would make voicing much easier to get right than the current trial-and-error method using the spinner.

2) Brightness : I see that there's already been extensive discussion about the tradeoff with CPU load depending on how this is implemented. I haven't got any new thoughts, but agree it would useful if feasible. EG one of my organs has a shrieking 2' rank that currently I can only make bearable by reducing the volume. It would be great if I could do some harmonic adjustment whilst voicing.

3) Flexible wind : beyond my expertise, but sounds like it would enhance the realism of organs - but at what CPU cost?
